genesyscloud/idp_adfs/resource_genesyscloud_idp_adfs_schema.go

```diff
@@ -72,10 +71,9 @@ func ResourceIdpAdfs() *schema.Resource {
 				Type:        schema.TypeString,
 			},
 			`slo_binding`: {
-				Description:  `Valid values: HTTP Redirect, HTTP Post`,
-				Optional:     true,
-				Type:         schema.TypeString,
-				ValidateFunc: validation.StringInSlice([]string{`urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect`, `urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST`}, false),
+				Description: `Valid values: HTTP Redirect, HTTP Post`,
```

Comment:
you can modify the description here too.. its no longer just HTTP Redirect, HTTP Post`,